homeassistant/components/camera/ffmpeg.py

```diff
@@ -32,7 +32,7 @@
 async def async_setup_platform(hass, config, async_add_entities,
                                discovery_info=None):
     """Set up a FFmpeg camera."""
-    if not await hass.data[DATA_FFMPEG].async_run_test(config.get(CONF_INPUT)):
+    if not hass.data[DATA_FFMPEG].async_run_test(config.get(CONF_INPUT)):
```

Comment:
This change makes no sense. `async_run_test` is an async function. It needs to be awaited to execute. By calling it but not awaiting it, you're ignoring the call. 

If this call is not wanted, it should be removed completely. Not just break the execution.

Reply:
I'm merely submitting the revert of something that caused an issue.

Reply:
If you are unable to address my comments, we should close this PR.

Reply:
And leave an issue that was caused by the original PR? Have at it.

Reply:
So then it should be fine to remove lines 35 and 36 altogether.

Reply:
Removed the function and tested.  No observable issues.

Reply:
But `async_run_test` serves a purpose. Based on [the docs](https://www.home-assistant.io/components/ffmpeg/), it:

> Check if input is usable by ffmpeg.

Which I assume means - it validates that ffmpeg is installed, runs and hasn't been passed any invalid parameters. By removing this code, you're effectively ignoring the `run_test` configuration value on the `ffmpeg` platform.

It might be better to change the default `run_test` parameter value from `true` to `false` [here](https://github.com/home-assistant/home-assistant/blob/2a3004d0dad396e53d53713270a1e2953a81099f/homeassistant/components/ffmpeg.py#L43) if it's proving problematic for people? Or maybe just instructing impacted users to reconfigure the `ffmpeg` platform themselves?

For what it's worth, my ffmpeg cameras work fine with `run_test` on. Maybe `run_test` is actually picking up on real issues on some people's installs?

Reply:
That's why I only removed the await in this PR which was how it was originally. I don't know this integration and only wanted to get it back to a usable state. Then we can figure out what is best. It's worse now. So getting it in a better state is all I wanted to do.